packages/core/src/core/communication.ts

```diff
@@ -86,13 +87,17 @@ export async function validateSession(
     });
 }
 
```

Comment:
Maybe we should remove the repositories parsing in the `validateSession` function no ?

Reply:
Also we can check that this function is still use in the code as we remove one of the usage :)

Reply:
It's still use to check actual user authentication so I kept it and repositories still exist in that context, it just work differently so for now I didn't change that behavior but like I told you I want to get rid of this validate endpoint in a way or another. It'll obviously mean some update on the auth server since SM still need it for other reasons